flattentool/__init__.py

```diff
@@ -208,14 +208,15 @@ def unflatten(input_name, base_json=None, input_format=None, output_name=None,
         base[root_list_path] = list(result)
 
     if xml:
+        xml_root_path = base_configuration.get('XMLRootPath', 'iati-activities')
         if output_name is None:
             if sys.version > '3':
-                sys.stdout.buffer.write(toxml(base))
+                sys.stdout.buffer.write(toxml(base, xml_root_path))
```

Comment:
Could we call this `xml_root_tag` instead, as it's not really a path?

Reply:
sure!

Reply:
Fixed in 23dcdb4 and cada684.